packages/core/src/components/button/button.scss

```diff
@@ -49,19 +53,27 @@
     &.-action--secondary {
       border-color: helpers.color('border-action');
 
-      &:hover,
-      &:active {
-        border-color: helpers.color('border-action-hover');
+      /* stylelint-disable max-nesting-depth */
+      @media (hover: hover) {
+        &:hover,
+        &:active {
```

Comment:
I don't think we should wrap `:active` here too, we probably still want that style to apply in touch devices that don't support `hover`, i.e. iOS